packages/utils/src/arrayDistinct.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Helper function to filter only distinct elements of an array
+ */
+export const arrayDistinct = <T>(txid: T, index: number, self: T[]) => self.indexOf(txid) === index;
```

Comment:
`txid` is not good name here, since this should be universal array distinct function

Reply:
Oops, I forgot it there from its previous location. Will fix it.